### PR TITLE
Use _log_ instead of echo for logging messages

### PR DESCRIPTION
**Pull request by ntrrgc**
This makes it easier to disable logging messages with new --quiet options or to use syslog/journald in the future.

### CODE REVIEW

File: utilities/host-setup-tasks.sh [hunk 1]
```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
 
+[ -z "${application_ready}" ] && { echo "[FATAL] You need to source 'utilities/application.sh' before sourcing this script."; return 1; }
```
Comment (nikolaszimmermann) [inline utilities/host-setup-tasks.sh:3]: Is this addition intentional?

File: scripts/container-only/wkdev-setup-devhelp [hunk 2]
```diff
@@ -65,18 +65,20 @@ run() {
...
-    printf "  Books:\t"
-    for dir in "${docs_dir}"/*; do
-        echo -n " $(basename "${dir}")"
-        ln --symbolic --force "${dir}" "${books_dir}/$(basename "${dir}")"
-    done
-    echo ""
-
-    echo ""
-    echo "Simply run 'devhelp' to access the documentation."
+    _log_ "$(
```
Comment (nikolaszimmermann) [inline scripts/container-only/wkdev-setup-devhelp:68]: That seems like the only non-mechanical change? Did you check the script still works? Code wise it looks fine.

**Review by nikolaszimmermann — APPROVED** [review]
I assume you checked that the log method is available in all scripts that use it now?

Looks good, just a few questions.

File: utilities/host-setup-tasks.sh (hunk 1 above)
Comment (ntrrgc) [reply utilities/host-setup-tasks.sh:3]: Yes, as (1) it reflects usage as far as I could find, (2) `_log_` is defined in `application.sh`, so it would fail anyway without it.

File: scripts/container-only/wkdev-setup-devhelp (hunk 2 above)
Comment (ntrrgc) [reply scripts/container-only/wkdev-setup-devhelp:68]: I wasn't even aware of this command existing before, but I tested it now and it seems to work.

File: utilities/host-setup-tasks.sh (hunk 1 above)
Comment (nikolaszimmermann) [reply utilities/host-setup-tasks.sh:3]: Good.

**Comment by ntrrgc** [comment]
> I assume you checked that the log method is available in all scripts that use it now?

I tried, hopefully didn't miss any.
